CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Change Log
 
+## [0.24.0](https://github.com/eclipse/lemminx/milestone/37?closed=1) (January 31, 2023)
+
+### Enhancements
```

Comment:
- https://github.com/eclipse/lemminx/issues/1405 (it wasn't labelled correctly so it wasn't picked up)